src/client/login/Login.js

```diff
@@ -5,19 +5,31 @@ import styles from './Login.css';
 export default class Login extends Component {
   constructor(props){
     super(props);
+    this.state = {
+      username: ''
+    };
     this.onSubmit = this.onSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
   }
+  handleChange(event){
+    this.setState ({
+      username: event.target.value
+    });
+  }
+
   onSubmit(e){
-    e.preventDefault();
-    var username = document.getElementById("inputfield").value;
+    var username = this.state.username;
```

Comment:
Can be `const`. You can also use object destructuring here, it's a common pattern used for state and props

`const { username } = this.state`